app/javascript/components/enrollment/Enrollment.js

```diff
@@ -166,7 +166,7 @@ class Enrollment extends React.Component {
         }
       })
       .catch(err => {
-        reportError(err);
+        reportError(err?.response?.data?.error ? err.response.data.error : err, false);
```

Comment:
Should this have been reverted?

Reply:
yes it should have - fixed here: https://github.com/SaraAlert/SaraAlert/pull/845/commits/b8f181a7a44a4ba03b3980e864a643bfb51c02ad